blocks/graph.py

```diff
@@ -244,12 +244,20 @@ def replace(self, replacements):
 
         return ComputationGraph(outputs_cur)
 
-    def get_theano_function(self, additional_updates=None):
-        """Create Theano function from the graph contained."""
+    def get_theano_function(self, additional_updates=None, **kwargs):
+        """Create Theano function from the graph contained.
+
+        Parameters
+        ----------
+        kwargs : key word arguments to theano.function
```

Comment:
Please see https://github.com/mila-udem/blocks/blob/master/blocks/bricks/sequence_generators.py#L814 for the proper way to add `kwargs` to `Parameters`.